VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -228,7 +224,7 @@ private void updateDataModel(){
         PrefManager.UserPrefManager userPrefManager = new PrefManager.UserPrefManager(MainApplication.instance());
         EnumSet<BlockType> types =  userPrefManager.isUserPrefVideoModel() ?
                              EnumSet.of(BlockType.VIDEO) : EnumSet.allOf(BlockType.class);
-        ((CourseComponent) selectedUnit.getRoot()).fetchAllLeafComponents(leaves, types);
+        (selectedUnit.getRoot()).fetchAllLeafComponents(leaves, types);
```

Comment:
Might as well get rid of the extra parenthesis surrounding `selectedUnit.getRoot()`.